### ISSUE TITLE
Problems compiling

### ISSUE THREAD

**Issue by mboisson** [cjcoats/ioapi-3.2#4]
Hi,
I am trying to understand the overly complicated build procedure for this project, and I am hitting all sorts of errors, such as

```
$ make
(cd /home/mboisson/git/easybuild-easyconfigs/easybuild/easyconfigs/i/IOAPI/ioapi-3.2-2020111/ioapi   ; make fixed_src)
make[1] : on entre dans le répertoire « /home/mboisson/git/easybuild-easyconfigs/easybuild/easyconfigs/i/IOAPI/ioapi-3.2-2020111/ioapi »
make[1]: rien à faire pour « fixed_src ».
make[1] : on quitte le répertoire « /home/mboisson/git/easybuild-easyconfigs/easybuild/easyconfigs/i/IOAPI/ioapi-3.2-2020111/ioapi »
make: ***  Aucune règle pour fabriquer la cible « /home/mboisson/git/easybuild-easyconfigs/easybuild/easyconfigs/i/IOAPI/ioapi-3.2-2020111/ioapi/Makefile », nécessaire pour « configure ». Arrêt.
``` 

```
$ make BIN=Linux2_x86_64ifort BASEDIR=$PWD
(cd /home/mboisson/git/easybuild-easyconfigs/easybuild/easyconfigs/i/IOAPI/ioapi-3.2-2020111/ioapi   ; make fixed_src)
make[1] : on entre dans le répertoire « /home/mboisson/git/easybuild-easyconfigs/easybuild/easyconfigs/i/IOAPI/ioapi-3.2-2020111/ioapi »
make[1]: rien à faire pour « fixed_src ».
make[1] : on quitte le répertoire « /home/mboisson/git/easybuild-easyconfigs/easybuild/easyconfigs/i/IOAPI/ioapi-3.2-2020111/ioapi »
(cd /home/mboisson/git/easybuild-easyconfigs/easybuild/easyconfigs/i/IOAPI/ioapi-3.2-2020111/ioapi   ;  sed -e 's|IOAPI_BASE|/home/mboisson/git/easybuild-easyconfigs/easybuild/easyconfigs/i/IOAPI/ioapi-3.2-2020111|' -e 's|LIBINSTALL||' -e 's|BININSTALL||' -e 's|IOAPI_DEFS||' -e 's|NCFLIBS|-lnetcdff -lnetcdf|' -e 's|MAKEINCLUDE|include /home/mboisson/git/easybuild-easyconfigs/easybuild/easyconfigs/i/IOAPI/ioapi-3.2-2020111/ioapi/Makeinclude|' -e 's|PVMINCLUDE|include  |' < Makefile..sed > Makefile )
/cvmfs/soft.computecanada.ca/gentoo/2020/bin/sh: Makefile..sed: Aucun fichier ou dossier de ce type
``` 

Do you have any plan to use some common build tool, such as 
Autoconf https://www.gnu.org/software/autoconf/
or
CMake https://cmake.org/

**Comment by cjcoats** [comment]
Are you following the instructions here:

https://cjcoats.github.io/ioapi/AVAIL.html#build
Availability/Download of the BAMS/Models-3 I/O API<https://cjcoats.github.io/ioapi/AVAIL.html#build>
Build and installation instructions for I/O API Versions 3.0, 3.1, and 3.2 As of this writing (June 2019), I/O API Versions 2.2 and earlier are obsolete; 3.0 and 3.1 obsolescent, in critical-fixes-only mode; and 3.2 is the current stable/production version.
cjcoats.github.io
...or are you doing something else?

Note that you don't need to do the  make fixed_src  part of the build unless you're doing un-authorized things like changing fundamental parameter-values; the distributed  fixed_src/*.EXT  files are read-only to try to prevent the sort of hard-to-diagnose messes that people have put themselves into by trying to make such changes.


Carlie J. Coats, Jr., Ph.D.                    cjcoats@email.unc.edu
Senior Software Engineer                   I/O API Author/Maintainer
Center for Environmental Modeling for Policy Development,
UNC Institute for the Environment                     www.ie.unc.edu
100 Europa Dr., Suite 490 Rm 4051 / Campus Box 1105     919.843.5951
Chapel Hill, NC 27599-1105                          Fax 919.966.9920

________________________________

**Comment by mboisson** [comment]
I was trying to decipher the correct installation instructions from that very long page, yes, but I could not (and I have installed hundreds of software packages on Linux in the past). 

I ended up using EasyBuild, with this recipe https://github.com/ComputeCanada/easybuild-easyconfigs/blob/computecanada-master/easybuild/easyconfigs/i/ioapi/ioapi-3.2-2020111-GCC-9.3.0.eb

Thankfully, somebody else figured it out for me and created an EasyConfig for it. Hopefully, it keeps working for future versions (which apparently, change even though the version number remains the same... 🤷‍♂️ )
